src/repository/base.repository.js

```diff
@@ -63,6 +63,17 @@ class BaseRepository {
     return model.update(fields, { where: options });
   }
 
+  /**
+   * @static
+   * @param {object} data - data of the article
+   * @param {object} options - column options
+   * @returns {object} - returns an database object
+   * @memberof BaseRepository
+   */
```

Comment:
Use the update field already available in the base repository class

Reply:
Thank you for this feedback. 
The `update` method there is not returning the object after performing the update operation. 
So, the new `updateField` method is a product of `pair-programming` with Princewill where we came up with the new `updateField` method to be able to return the updated object.